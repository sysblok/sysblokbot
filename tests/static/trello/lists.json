[
  {
    "id": "list_0",
    "name": "Идея для статьи",
    "closed": false,
    "pos": 794623,
    "softLimit": null,
    "idBoard": "board_id",
    "subscribed": false
  },
  {
    "id": "list_1",
    "name": "Готовая тема - автор, бери!",
    "closed": false,
    "pos": 811007,
    "softLimit": null,
    "idBoard": "board_id",
    "subscribed": false
  },
  {
    "id": "list_2",
    "name": "Уже пишу (не забудь указать дату и автора!)",
    "closed": false,
    "pos": 818175,
    "softLimit": null,
    "idBoard": "board_id",
    "subscribed": false
  },
  {
    "id": "list_3",
    "name": "Редактору",
    "closed": false,
    "pos": 841215,
    "softLimit": null,
    "idBoard": "board_id",
    "subscribed": false
  },
  {
    "id": "list_4",
    "name": "На редактуре на след.неделю (не забудь тег рубрики!)",
    "closed": false,
    "pos": 864255,
    "softLimit": null,
    "idBoard": "board_id",
    "subscribed": false
  },
  {
    "id": "list_5",
    "name": "Отредактировано впрок",
    "closed": false,
    "pos": 867327,
    "softLimit": null,
    "idBoard": "board_id",
    "subscribed": false
  },
  {
    "id": "list_6",
    "name": "Финальная проверка и отбор (Главреду)",
    "closed": false,
    "pos": 870399,
    "softLimit": null,
    "idBoard": "board_id",
    "subscribed": false
  },
  {
    "id": "list_7",
    "name": "Отобрано для публикации на неделю (Корректору)",
    "closed": false,
    "pos": 870911,
    "softLimit": null,
    "idBoard": "board_id",
    "subscribed": false
  },
  {
    "id": "list_8",
    "name": "Готово для верстки (Выпускающему)",
    "closed": false,
    "pos": 871423,
    "softLimit": null,
    "idBoard": "board_id",
    "subscribed": false
  },
  {
    "id": "list_9",
    "name": "Долгий Ящик",
    "closed": false,
    "pos": 1155071,
    "softLimit": null,
    "idBoard": "board_id",
    "subscribed": false
  }
]