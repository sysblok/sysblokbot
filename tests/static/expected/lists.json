[
  {
    "id": "list_0",
    "name": "Идея для статьи",
    "idBoard": "board_id"
  },
  {
    "id": "list_1",
    "name": "Готовая тема - автор, бери!",
    "idBoard": "board_id"
  },
  {
    "id": "list_2",
    "name": "Уже пишу (не забудь указать дату и автора!)",
    "idBoard": "board_id"
  },
  {
    "id": "list_3",
    "name": "Редактору",
    "idBoard": "board_id"
  },
  {
    "id": "list_4",
    "name": "На редактуре на след.неделю (не забудь тег рубрики!)",
    "idBoard": "board_id"
  },
  {
    "id": "list_5",
    "name": "Отредактировано впрок",
    "idBoard": "board_id"
  },
  {
    "id": "list_6",
    "name": "Финальная проверка и отбор (Главреду)",
    "idBoard": "board_id"
  },
  {
    "id": "list_7",
    "name": "Отобрано для публикации на неделю (Корректору)",
    "idBoard": "board_id"
  },
  {
    "id": "list_8",
    "name": "Готово для верстки (Выпускающему)",
    "idBoard": "board_id"
  },
  {
    "id": "list_9",
    "name": "Долгий Ящик",
    "idBoard": "board_id"
  }
]